solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-10.adoc

```diff
@@ -35,6 +35,14 @@ Minimum Java version for Solr 10.x is Java 21.
 The Solr CLI has gone through some significant renovations to reduce technical debt, and now functions more consistently and predictably. Most notably, CLI commands now use unix-style options, e.g. `--help` instead of `-help`.
 Users are urged to review all use of the `bin/solr` command, such as in scripts and documentation, to ensure that the correct options are being used.
 
+Some key changes that you may run into are:
+
+ * Commands that interact with Solr now all use `--solr-url` (or `-s`) plus a `--name` (or `-c`) to specify the Solr to interact with.
+ * You may use `--zk-host` (or `-z`) in place of the `--solr-url` option to lookup the Solr host.
+ * Basic Authentication support has been added to bin/solr via the `--credentials` (or `-u`) option.
+
```

Comment:
Believing that we will also remove the deprecated options in 10, this could also be mentioned:

```suggestion
 * Some short and single-letter options have been removed to avoid conflicts or in favor to other options.
 
```